getTestRule.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
 const util = require('util');
-// eslint-disable-next-line node/no-unpublished-require -- Avoid auto-install of `stylelint` peer dependency.
-const { lint } = require('stylelint');
+
+// @ts-expect-error -- TS2614: Module '"stylelint"' has no exported member 'lint'. Did you mean to use 'import lint from "stylelint"' instead?
```

Comment:
[note] This TS error should be resolved when we rewrite the current code to ESM. I want to keep the code as-is.